apps/backend/alembic/versions/66cc68f4e112_fix_configuration_snapshots_id_column_.py

```
@@ -0,0 +1,67 @@
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import UUID
+
+
+# revision identifiers, used by Alembic.
+revision: str = '66cc68f4e112'
+down_revision: Union[str, Sequence[str], None] = 'af4ba97c1822'
```

Ruff (0.12.2)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)